tests/integration/test_networking.py

```diff
@@ -23,6 +23,7 @@
 async def test_services_have_matching_labels(
     kube_client: AsyncClient,
     generated_data: ESSData,
+    matrix_stack,
```

Comment:
matrix_stack fixture has `autouse` enabled. Do we really need this ? 


```
@pytest.fixture(autouse=True, scope="session")
async def matrix_stack(
```

Reply:
This came about as a result of e.g. https://github.com/element-hq/ess-helm/actions/runs/19760559262/job/56621295246. That test has `matrix_stack` but doesn't have `prometheus_operator_crds` which is another `autouse=True` CRD. The only way I could see a complete failure of monitored pods but all monitorable pods being present is if the `ServiceMonitor` CRD wasn't yet present in cluster, which makes the reliance on `autouse=True` highly suspect.

I can try dropping these and just keeping the `prometheus_operator_crds` fixture being an explicit dependency of the chart